go/Makefile

```diff
@@ -15,4 +18,24 @@ clean:
 	rm -f pkg/sdk/optimism/abis/*.go
 	rm -f pkg/sdk/ethereum/abis/*.go
 
-test :; go test github.com/NovaSubDAO/nova-sdk/go/...
+test: build
+	@echo "Starting Anvil..."
+	@{ \
+		anvil --fork-url $(OPT_RPC_ENDPOINT) --port $(ANVIL_PORT) & echo $$! > $(ANVIL_PID_FILE); \
+		# Function to clean up Anvil process \
+		cleanup() { \
+			echo "Stopping Anvil..."; \
+			if [ -f $(ANVIL_PID_FILE) ]; then \
+				kill $$(cat $(ANVIL_PID_FILE)); \
```

Comment:
this command should be enough to kill the process? why rm the file as well below?